django/core/management/commands/inspectdb.py

```diff
@@ -269,19 +269,28 @@ def get_meta(self, table_name, constraints, column_to_field_name):
         to the given database table name.
         """
         unique_together = []
+        indexes = []
         for index, params in constraints.items():
+            fields = [column_to_field_name[c] for c in params['columns']]
             if params['unique']:
                 columns = params['columns']
                 if len(columns) > 1:
                     # we do not want to include the u"" or u'' prefix
                     # so we build the string rather than interpolate the tuple
-                    tup = '(' + ', '.join("'%s'" % column_to_field_name[c] for c in columns) + ')'
+                    tup = '(' + ', '.join("'%s'" % f for f in fields) + ')'
                     unique_together.append(tup)
+            elif params['index']:
+                indexes.append("models.Index(fields=[%s], name='%s')" % (', '.join("'%s'" % f for f in fields), index))
```

Comment:
I'm not sure about hardcoding `models.Index` here. I'd think we'd at least want to add a comment if the type of the introspected index doesn't match.